CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+# The Ruby Community Conduct Guideline 
+
+## We have picked the following conduct guideline based on an early proposed draft of the PostgreSQL CoC, for Ruby developers community for safe, productive collaboration. 
+
+### This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the Ruby community. It applies to all “collaborative space”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
```

Comment:
These can be just a paragraph, no heading needed. Also replace "Ruby community" with "Bemuse project".